src/pages/en/utol/lecturers/assignments/index.mdx

```diff
@@ -7,6 +7,273 @@ breadcrumb:
+     - [Move from the "Assignment Check assignment submission status" screen](#go-to-evaluate-3)
+2. After the "Register/Edit assignment feedback" screen opens, add "Grade", "Feedback comments" and upload the "Feedback file", followed by clicking the "Register" button.
+   ![課題フィードバック登録/編集](./12-evaluate-on-utol.png){:.center}
+3. When you click the "Prev" button located at the bottom left of the screen, or the "Next" button at the bottom right, you can move on to evaluating the previous/next enrolled student.
+
+### Evaluating by Downloading the Format
+{:#evaluate-with-format}
+
+1. Open the "[Checking the Submission Status of Each Assignment](#submission-status-per-assignment)" screen.
+2. Download a format from the "Submission Status" field. The type of download file varies depending on whether a "Feedback file" is attached.
+
+- If you don't need a "Feedback file" (both "Grade" and "Feedback comments" can be text only).
+  1. Click on the "Download Format" button (in blue text) in the row of "Comment and Grade".
+  2. When you are asked to set up a password for the Excel file, enter any password, and click on the "Download" button again.
+  3. A file in Excel format (`.xlsx` format) will be downloaded.
+- If you need to attach a "feedback file"
+  1. Click on the "Download zip format" button (in blue text) in the row of "Comment, Grade and Feedback file (zip file)".
+  2. When you are asked to set up a password for the ZIP file, enter any password, and click on the "Download" button again.
+  3. A file in ZIP format (`zip` format) will be downloaded.
+
+Enter and upload the evaluation according to the format of the downloaded file, following the steps below.
+
+#### For Excel Files
+
+1. Open the downloaded Excel file (`.xlsx` format).
+2. Enter "Grade", "Comment" and save the file.
+   - Please do not change the file name at this time.
+3. Move back to the "[Checking the Submission Status of Each Assignment](#submission-status-per-assignment)" screen.
+4. Click on the "Reference" button in the row of "Comments and Grade" in the "Submission Status" field and select the Excel file you have just entered.
+5. Click on the "Upload" button.
+6. When you are asked to enter the read password for the file you are uploading, enter the password you set when downloading and press the "Upload" button again.
+
+#### For ZIP Files
+
+1. Unzip the downloaded ZIP file (`.zip` format) and open the Excel file (`.xlsx` format) located in the extracted folder.
+2. Enter "Grade", "Comment" and save the file.
+   - Please do not change the file name at this time.
+3. Place the feedback file in the "Feedback" folder located in the extracted folder and enter the file name in the "File name" field of the Excel file.
+   - e.g.: If the feedback file is for enrolled student A, it will be "`Feedback_A.pdf`". Place "`Feedback_A.pdf`" in the "Feedback" folder, and enter the "`Feedback_A.pdf`" in the "File name" field in the row for enrolled student A.
```

Comment:
theはなくて良い気がします．
```suggestion
   - e.g.: If the feedback file is for enrolled student A, it will be "`Feedback_A.pdf`". Place "`Feedback_A.pdf`" in the "Feedback" folder, and enter "`Feedback_A.pdf`" in the "File name" field in the row for enrolled student A.
```